src/comprehensions.py

```diff
@@ -11,13 +11,17 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
+for i in range(5):
+    y.append(1+i)
```

Comment:
You are using for loops, you should be implementing the logic inside the square brackets since it has to be a list comprehension